src/app/tests/suites/certification/Test_TC_TSTAT_2_1.yaml

```diff
@@ -231,16 +231,6 @@ tests:
               minValue: 0
               maxValue: 100
 
-    - label: "Step 11: Read HVACSystemTypeConfiguration attribute from the DUT"
```

Comment:
Because the test was looking for an attribute that was marked D in the spec for 1.0.  We don't need it in the SDK, and it probably never should've been there, and it's no longer in the XML as a result.  So to pass the test we decided to remove it since it isn't actually a certification requirement.

Reply:
We can't remove the attribute from the XML without breaking shipped public APIs or doing something complicated to not break them.  I strongly suggest leaving it in the XML for now for expediency and filing an issue on figuring out a way to remove it.

Reply:
ok, I'll restore the attribute and the unnecessary test since it could break API's.